src/dune

```
@@ -123,6 +123,11 @@
 )
 
 (env
+  (release
+    (flags (:standard -warn-error -A -w -unused-var-strict -w +9)) ; https://dune.readthedocs.io/en/stable/faq.html#how-to-make-warnings-non-fatal
```

An extra warning has been added to the other cases meanwhile:
```suggestion
    (flags (:standard -warn-error -A -w -unused-var-strict -w -unused-functor-parameter -w +9)) ; https://dune.readthedocs.io/en/stable/faq.html#how-to-make-warnings-non-fatal
```